service/provider/src/main/java/gov/va/vro/service/provider/bip/service/BipApiService.java

```diff
@@ -138,10 +128,10 @@ public List<ClaimContention> getClaimContentions(long claimId) throws BipExcepti
       HttpEntity<Map<String, String>> httpEntity = new HttpEntity<>(headers);
       ResponseEntity<String> bipResponse =
           restTemplate.exchange(url, HttpMethod.GET, httpEntity, String.class);
-      if (HttpStatus.OK.equals(bipResponse.getStatusCode())) {
+      if (HttpStatus.OK == bipResponse.getStatusCode()) {
         BipContentionResp resp = mapper.readValue(bipResponse.getBody(), BipContentionResp.class);
         return resp.getContentions();
-      } else if (HttpStatus.NO_CONTENT.equals(bipResponse.getStatusCode())) {
+      } else if (HttpStatus.NO_CONTENT == bipResponse.getStatusCode()) {
```

Comment:
Why compare object refs instead of the value in this case?

Reply:
They are enums.